tests/kafkatest/tests/core/log_dir_failure_test.py

```diff
@@ -84,20 +84,25 @@ def __init__(self, test_context):
         self.num_consumers = 1
 
     def setUp(self):
-        self.zk.start()
+        if self.zk:
+            self.zk.start()
 
     def min_cluster_size(self):
         """Override this since we're adding services outside of the constructor"""
         return super(LogDirFailureTest, self).min_cluster_size() + self.num_producers * 2 + self.num_consumers * 2
 
-    @cluster(num_nodes=9)
-    @matrix(bounce_broker=[False, True], broker_type=["leader", "follower"], security_protocol=["PLAINTEXT"])
-    def test_replication_with_disk_failure(self, bounce_broker, security_protocol, broker_type):
+    @cluster(num_nodes=8)
+    @matrix(bounce_broker=[False, True], broker_type=["leader", "follower"], security_protocol=["PLAINTEXT"], metadata_quorum=[quorum.zk])
+    @cluster(num_nodes=7)
+    @matrix(bounce_broker=[False, True], broker_type=["leader", "follower"], security_protocol=["PLAINTEXT"], metadata_quorum=[quorum.combined_kraft])
```

Comment:
Unsure about the value of repeating the tests for `quorum.combined_kraft`, this mode isn't recommended for production, it's easier to cover it in java integration tests and makes this suite slower.

Reply:
I agree with dropping the combined test here. I'm not sure I would ever suggest running JBOD in combined mode. A separate process for broker and controller on the same node would be better.